packages/fxa-auth-server/scripts/transfer-users/apple.js

```diff
@@ -198,32 +200,43 @@ export class AppleUser {
 
   async transferUser(accessToken) {
     await this.exchangeIdentifiers(accessToken);
-    await this.createUpdateFxAUser(this.appleUserInfo);
+    if (!this.err) {
+      await this.createUpdateFxAUser(this.appleUserInfo);
+    }
     this.saveResult();
     console.log(`Transfer complete: ${this.transferSub} ${this.success}`);
   }
   
   saveResult() {
-    const appleEmail = this.appleUserInfo.email;
-    const fxaEmail = this.accountRecord.email;
-    const uid = this.accountRecord.uid; // Newly created uid
-    const transferSub = this.transferSub;
-    const success = this.success;
-    const err = (this.err && this.err.message) || '';
-
-    this.log.notifyAttachedServices(
-      'appleUserMigration', {},
-      {
-        uid,
-        appleEmail,
-        fxaEmail,
-        transferSub,
-        success,
-        err,
-      },
-    );
-    const line = `${transferSub},${uid},${fxaEmail},${appleEmail},${success},${err}`;
-    this.writeStream.write(line + '\n');
+    if (!this.success) {
+      console.log(`Failed to transfer ${this.transferSub}`);
```

Comment:
Should remove if this is just for debugging.

Reply:
I'll keep this in, SRE ran script a couple ways so it was useful to monitor this and the outfile file.